modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Working with mount namespace encapsulation
+
+As an OpenShift system administrator, you want `systemd` to consume less resources so that you can run more workloads on my system. You want to enable and disable this setting, and verify whether encapsulation is in effect.
+
+== Enabling and disabling encapsulation
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have created the `MachineConfig` object.
```

Comment:
Which object is the pre-requisite here?

Reply:
I was confused by this too.  To enable this feature you can just create a brand new MachineConfig object without having created one in the past.

Reply:
Done.